Bottom: 90383e45241d33a9e9f5e053e5b748aff1939de5
Top:    284ec966d1840830047e956b6d838487606af4c7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-29 15:45:29 -0500

Refresh of buildbot-worker-ntel

---
diff --git a/docker/buildbot-worker-ntel/buildbot/buildbot.tac b/docker/buildbot-worker-ntel/buildbot/buildbot.tac
index 058c6a08..37012ab4 100644
--- a/docker/buildbot-worker-ntel/buildbot/buildbot.tac
+++ b/docker/buildbot-worker-ntel/buildbot/buildbot.tac
@@ -18,7 +18,7 @@ application.setComponent(ILogObserver, FileLogObserver(sys.stdout).emit)
 buildmaster_host = os.environ.get("BUILDBOT_MASTER", 'localhost')
 port = int(os.environ.get("BUILDBOT_WORKER_PORT", 9989))
 workername = os.environ.get("BUILDBOT_WORKER_NAME", 'docker')
-passwd = os.environ.get("BUILDBOT_WORKER_PASSWORD")
+passwd = os.environ.get("BUILDBOT_WORKER_PASS")
 
 # delete the password from the environ so that it is not leaked in the log
 blacklist = os.environ.get("BUILDBOT_ENVIRONMENT_BLACKLIST", "BUILDBOT_WORKER_PASSWORD").split(',')
